src/applications/gi/components/vet-tec/EybVetTecApprovedPrograms.jsx

```diff
@@ -0,0 +1,110 @@
+        ? this.props.selectedProgram
+        : this.props.preSelectedProgram;
+
+    if (programs && programs.length) {
+      const programRows = programs.map((program, index) => {
+        const programLength =
+          isPresent(program.lengthInHours) && program.lengthInHours !== '0'
+            ? `${program.lengthInHours} hours`
+            : 'TBD';
+        const tuition = isPresent(program.tuitionAmount)
+          ? formatCurrency(program.tuitionAmount)
+          : 'TBD';
+        const checked =
+          selectedProgram &&
+          program.description.toLowerCase() === selectedProgram.toLowerCase();
+        return (
+          <tr key={index}>
+            <td className="vads-u-padding-left--0 vads-l-grid-container">
+              <div className="program-description vads-l-row">
+                <div className="vads-l-col--2 checked-program">
+                  <i
+                    className={
+                      checked
+                        ? 'fas fa-check vads-u-padding-right--0p5 vads-u-color--green'
+                        : 'vads-u-padding-right--2p5'
+                    }
+                  />
+                </div>
+                <div className="vads-l-col--10 ">
+                  {program.description}{' '}
+                  {checked ? (
+                    <span className="sr-only">selectedProgram</span>
+                  ) : (
+                    ''
+                  )}
+                </div>
+              </div>
+            </td>
+            <td className="vads-u-padding-y--0 program-length">
+              {programLength}
+            </td>
+            <td className="vads-u-padding-y--0">{tuition}</td>
+          </tr>
+        );
+      });
+
+      return (
+        <div className="vads-u-margin-top--2">
+          <span>Select a program below to view your estimated benefits.</span>
+          <table className="vet-tec-programs-table">
+            <colgroup>
+              <col className="name-col" />
+              <col className="hours-col" />
+              <col className="tuition-col" />
+            </colgroup>
+            <thead>
+              <tr>
+                <th id="program-name-header">Program name</th>
+                <th>Length</th>
+                <th>Tuition & Fees</th>
+              </tr>
+            </thead>
+            <tbody>{programRows}</tbody>
+          </table>
+        </div>
+      );
+    }
+    return (
+      <div>
+        <p>
+          Program data will be available for this provider soon.{' '}
+          <a
+            href={
+              '/education/about-gi-bill-benefits/how-to-use-benefits/vettec-high-tech-program/'
+            }
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            Learn more about VET TEC programs
+          </a>
+        </p>
+      </div>
+    );
+  }
+}
+
+const mapStateToProps = state => ({
+  selectedProgram: state.calculator.selectedProgram,
```

Comment:
Right now this file should be in containers because it's connected to the Redux store, but instead of moving it, can you have the parent pass in selectedProgram since that's the one thing mapStateToProps is being used for?

Reply:
Done and I also removed it from `VetTecEstimateYourBenefitsform.jsx`.